pr-data.csv

```diff
@@ -1423,10 +1423,10 @@ https://github.com/apache/flink,7542b56f2abb860f42a83c4687f6e38bb82b78c6,flink-t
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.runtime.harness.AggFunctionHarnessTest.testCollectAggregate,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.bundle.MapBundleOperatorTest.testSimple,ID,Rejected,https://github.com/apache/flink/pull/11268,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.ProcTimeBoundedStreamJoinTest.testProcTimeInnerJoinWithCommonBounds,ID,Rejected,https://github.com/apache/flink/pull/11287,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.RowTimeBoundedStreamJoinTest.testRowTimeFullOuterJoin,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/20
+https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-runtime-blink,org.apache.flink.table.runtime.operators.join.RowTimeBoundedStreamJoinTest.testRowTimeFullOuterJoin,ID,,,https://github.com/apache/flink/commit/ab70dcfa19827febd2c3cdc5cb81e942caa5b2f0;https://github.com/TestingResearchIllinois/idoft/issues/20
```

Comment:
I don't understand the intended change here. You're keeping exactly the same module/class/method and adding some commit link. Why?

Reply:
Sorry. Ill update that, i was trying to squash the previous commits.

Reply:
Any progress on squashing commits?